lamar/tasks/feature_extraction.py

```diff
@@ -147,4 +147,25 @@ class RetrievalFeatureExtraction(FeatureExtraction):
                 'preprocessing': {'resize_max': 640},
             }
         },
+        'cosplace': {
+            'name': 'cosplace',
+            'hloc': {
+                'model': {'name': 'cosplace'},
+                'preprocessing': {'resize_max': 640},
+            }
+        },
```

Comment:
https://github.com/cvg/Hierarchical-Localization/commit/d0e8494e66fa63fc663cf6408c9e2d057af919ac removes Cos/EigenPlaces but adds MegaLoc, which is much more robust (and has results on LaMAR).

Reply:
Thanks for the info, I'll go ahead and replace with MegaLoc and will also try that one on a currently private data split (to be released) to see where it stands (SALAD seems kinda good 😄 )